azure-pipelines.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           # run unit test and build executable
           # the set -e line is needed so that the unit tests failure would cause the job to fail properly
           set -e
-          go test -race -short -cover ./cmd ./common ./ste ./azbfs
+          go test -timeout 30m -race -short -cover ./cmd ./common ./ste ./azbfs
```

Comment:
>-timeout 30m [](start = 18, length = 12)

please leave a comment on why the 30 min is required.

Reply:
30 isn't _super_ necessary. It was more or less a sanity check. I'll set it to 15 and leave a note.